examples/player/simid_player.js

```diff
@@ -393,6 +418,20 @@ class SimidPlayer {
         .then(() => this.destroyIframeAndResumeContent_());
   }
 
+  /**
+   * Skips the ad and destroys the ad iframe.
+   * @param {StopCode} reason The reason the ad will stop.
+   */
+  skipAd() {
+    // The iframe is only hidden on ad stoppage. The ad might still request
+    // tracking pixels before it is cleaned up.
+    this.hideSimidIFrame_();
+    // Wait for the SIMID creative to acknowledge stop and then clean
+    // up the iframe.
+    this.simidProtocol.sendMessage(PlayerMessage.AD_SKIPPED, "Ad skipped by player")
```

Comment:
The ad skipped message doesn't have any additional data. You don't need to pass in anything to sendMessage besides the message type

Reply:
when tracing through the calls this message would be added to the args field, but sure I can get rid of it if it is unnecessary

Reply:
Yeah, you're right it would be added to the message, but since the spec doesn't talk about this, any creative implementation wouldn't know to look for it